src/tag/tag.js

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+import {getOverrideObject} from '../helpers/overrides';
+import type {PropsT, StatelessStateT} from './types';
+import {Action as StyledAction, Root as StyledRoot} from './styled-components';
+import {STYLE} from './constants';
+
+class StatelessTag extends React.Component<PropsT, StatelessStateT> {
```

Comment:
Can we just call this `Tag`? There's no stateful version so disambiguating is kind of pointless.

You can also leave out the StatelessStateT

Reply:
ok